src/Provider/GeocodeEarth/README.md

```diff
@@ -18,10 +67,9 @@ composer require geocoder-php/geocode-earth-provider
 
 ### API Documentation
 
-Geocode Earth uses the Pelias Geocoder under the hood. You can view it's [documentation here](https://github.com/pelias/documentation).    
-The base API endpoint is https://api.geocode.earth.
+You can view the complete [API documentation](https://geocode.earth/docs) on their website.
+The base API endpoint is `https://api.geocode.earth`.
```

Comment:
Why removing mention to Pelias ?

Reply:
True, there is some overlap with the documentation we publish for the open-source project at https://github.com/pelias/documentation and the docs we publish for the hosted service.

GeocodeEarth is a hosted *distribution of Pelias* from the authors of the project, it's not exactly the same as what you get when you download the FOSS software and run it yourself.

The documentation for downloading and installing Pelias is not relevant for users of Geocode Earth and likewise documentation about authentication and rate-limits is not relevant to users cloning the repos.

As the founder/maintainer of both I'd prefer if the correct documentation was linked to for each, although I'm not against cross-linking to https://github.com/geocoder-php/Geocoder/tree/master/src/Provider/Pelias if you think that is beneficial.

Reply:
Okay, I understand 👍 

Since our GeocodeEarth provider is based on our Pelias provider (see below), I would indeed keep a mention to our Pelias provider somewhere in the README.
https://github.com/geocoder-php/Geocoder/blob/4bf63c85196df24699a11e857f2256e6a7c36ce5/src/Provider/GeocodeEarth/GeocodeEarth.php#L23

PS: I didn't notice you are the founder of Pelias and GeocodeEarth ; thanks a lot for your awesome work ! 🤩

Reply:
Thanks for the kind words.

Here is a matrix of providers and the FOSS software they use to power their commercial business:

| Business        | FOSS           | Role |
| ------------- |:-------------:|:-------------:|
| [Geocode Earth](https://github.com/geocoder-php/geocode-earth-provider) | [Pelias](https://github.com/pelias/pelias) | Maintainer |
| [Mapzen](https://github.com/geocoder-php/Geocoder/tree/master/src/Provider/Mapzen) | [Pelias](https://github.com/pelias/pelias) | Previous Maintainer |
| [Open Route Service](https://github.com/geocoder-php/Geocoder/tree/master/src/Provider/OpenRouteService) | [Pelias](https://github.com/pelias/pelias) |
| [Graphhopper](https://github.com/geocoder-php/graphhopper-provider) | [Photon](https://github.com/komoot/photon) | Maintainer |
| [OpenCage](https://github.com/geocoder-php/open-cage-provider) | [Nominatim](https://github.com/osm-search/Nominatim) | Maintainer |
| [Mapbox](https://github.com/geocoder-php/mapbox-provider) | [Carmen](https://github.com/mapbox/carmen) | Maintainer |
| [LocationIQ](https://github.com/geocoder-php/locationiq-provider) | [Nominatim](https://github.com/osm-search/Nominatim) |

I would love to see other geocoding businesses providing attribution to the developers who build and maintain the software which powers their commercial business, particularly those which don't contribute back to the project.

I suspect the reason they don't do so is because they don't want users to know that there is a FOSS alternative available to their commercial service.

By providing Pelias for free (and by linking it from our business provider page) we are giving users full control over how they configure and host their geocoder, I think this is in the best interest of developers and the community at-large.

This does however put us at a disadvantage compared to a company like *cough* LocationIQ *cough* who do not attribute the authors and maintainers of the FOSS software which powers their platform (these guys go as far as [removing attribution from source code after forking repos](https://github.com/location-iq/leaflet-geocoder/pull/5/files)).

The nett effect of this is that all the costs of developing/maintaining/testing the software fall on the maintainers of the FOSS software while all the finances required to sustain maintenance of the software go to the commercial providers.

I'll go ahead and make that change, I think you're right, it's the best thing for users of this library to have informed choices about whether they would prefer someone hosted it for them (for a fee) or they hosted it themselves.

We have a steady stream of SaaS clones popping up powered by Pelias who compete directly against the authors and maintainers while passing support issues on to us:

> Ben Thompson wrote an excellent summary in his piece AWS, MongoDB, and the Economic Realities of Open Source almost two years ago. To summarize his summary: if you give your secret sauce away for free, and it gets popular enough, cloud providers will inevitably spin up competitive services using your very own code against you. They will ruthlessly, unapologetically, shamelessly bludgeon you with a rubber chicken of your own fashioning. They’ll take a dump in your front yard while your lawyer stands over your shoulder whispering, “nothing can be done.”

https://joemorrison.medium.com/death-of-an-open-source-business-model-62bc227a7e9b

Sorry for the rant 😆

Reply:
> Sorry for the rant 😆

Not at all, this is awesome information.
I wasn't aware that Mapzen, Graphhopper, OpenCage, Mapbox, and LocationIQ were based on FOSS software.
That should also allow use to "merge" some of our providers (like we did for Geocode Earth and Open Route Service, both extending the Pelias provider).

I'll also make sure to attribute properly the FOSS software begind the geocoding service in our README files.

Thanks for the link to the article. 👍

Reply:
![51p7yz](https://user-images.githubusercontent.com/738069/111084562-dff65f80-8577-11eb-9f03-23328f297c87.jpg)

Thanks!